src/MizarWidgets/SamplingWithFrameTrackInputWidget.cpp

```diff
@@ -22,6 +23,8 @@ SamplingWithFrameTrackInputWidgetBase::SamplingWithFrameTrackInputWidgetBase(QWi
   ui_->setupUi(this);
   QObject::connect(GetThreadList(), &QListWidget::itemSelectionChanged, this,
                    &SamplingWithFrameTrackInputWidgetBase::OnThreadSelectionChanged);
+  QObject::connect(GetFrameTrackList(), SIGNAL(currentIndexChanged(int)), this,
+                   SLOT(OnFrameTrackSelectionChanged(int)));
```

Comment:
Please use the new connect overload with function pointers. You were probably struggling because currentIndexChanged has two overloads. You can use `qOverload` to distinguish between them. Check out https://doc.qt.io/qt-6/qtglobal.html#qOverload for details and an example.